projects/govuk-chat/Makefile

```diff
@@ -0,0 +1,4 @@
+govuk-chat: bundle-govuk-chat
+	$(GOVUK_DOCKER) run $@-lite bin/rails db:setup
+	$(GOVUK_DOCKER) run $@-lite env RAILS_ENV=test bin/rails db:setup
+	$(GOVUK_DOCKER) run $@-lite yarn
```

Comment:
I don't think we yet have yarn installed (I noticed this wasn't part of the dockerfile)

Reply:
I think this is required for when we need to run `govuk-docker run govuk-chat-lite bundle exec rake`  locally. The package.json includes the `standard` linting command , which fails if we don't run yarn. I might be wrong, but couldn't make it work otherwise

```
"scripts": {
    "lint": "yarn run lint:js && yarn run lint:scss",
    "lint:js": "standard 'app/assets/javascripts/**/*.js'",
    "lint:scss": "stylelint app/assets/stylesheets/"
  },
```